Lib/base64.py

```diff
@@ -430,6 +430,7 @@ def b85encode(b, pad=False):
     # if the function is never called
     if _b85chars is None:
```

Comment:
It is enough to check `if _b85chars2 is None` here.

These checks are cheap, but they are performed at every `b85encode()` call. Lesser checks is better.

`a85encode()` should be updated too. And may be other functions if they initialize two or more globals.

Reply:
done